dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2/src/org/apache/cxf/transport/http/osgi/HTTPTransportActivator.java

```diff
@@ -0,0 +1,90 @@
+/**
```

Comment:
What changed here?

Reply:
On our code base for CXF 2.6.2 we have changes. However, with CXF 3.x whole design of this class is changed. None of our changes were compatible and none applied. Removing this file.